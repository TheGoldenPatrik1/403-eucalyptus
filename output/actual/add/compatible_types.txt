6
2.7
[1, 2, 3]
test test2
