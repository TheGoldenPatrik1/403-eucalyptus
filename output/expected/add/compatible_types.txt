6
2.7
[1, 2, 3]
test test2
{test=1, test2=2, test3=3}
[1, 2]
{test=1, test2=2}